### ISSUE TITLE
Profile Picture Upload 

### ISSUE THREAD

**Issue by ChaseWGregory** [Code-Pop/vue-mastery#59]
I signed up by the download cheat sheet button, when I uploaded a profile picture to my account, I was directed to another download of the cheat sheet.

**Comment by pieer** [comment]
What is the behavior expected here? I'm not sure to understand the problem

**Comment by ChaseWGregory** [comment]
I signed up for an account with the Download Cheat Sheet Button on the homepage. When I click, this automatically downloads the pdf to my computer and sends me an email. I clicked on Dashboard after that, then clicked to upload a profile picture. When I uploaded the photo, the download pdf action happened again and the pdf showed up in front of my screen as if I just downloaded it again. I now have 2 copies.
I dont need 2 copies. 
Then I was not redirected to do anything after this. Will be nice to be directed to Update more profile info or start a course. It just seemed like a dead end, I know Gregg wants everything to loop, no loose ends. I think this is one of them. 


--Chase Gregory
Vue Mastery Head of Marketing


"The Ultimate Learning Resource for Vue Developers."

www.VueMastery.com

--- original message ---
On Thu, Mar 15, 2018 at 01:29 pm, <notifications@github.com> Pierre Schweiger wrote:


What is the behavior expected here? I'm not sure to understand the problem

—

You are receiving this because you authored the thread.

Reply to this email directly, view it on GitHub, or mute the thread.
--- end of original message ---
